win32ss/printing/base/winspool/precomp.h

```diff
@@ -40,4 +40,6 @@ SPOOLER_HANDLE, *PSPOOLER_HANDLE;
 // main.c
 extern HANDLE hProcessHeap;
 
+extern BOOL UnicodeToAnsiInPlace(PWSTR pwszField);
```

Comment:
Nitpick: Please add a comment `// utils.c` here to keep things consistent with the rest of the file.

Reply:
Done. Thanks.